components/ContributorLead.vue

```diff
@@ -1,9 +1,5 @@
 <template>
-    <v-card
-    class="mx-auto"
-    max-width="344"
-    outlined
-    >
+  <v-card class="mx-auto" max-width="344" outlined>
```

Comment:
Was this file changed from prettify? It doesn't seem to have anything to do with logging.

Reply:
I didn't change manually, I guess it changed when I ran `yarn lint`. Let me check.

Reply:
Ok cool, if it's automatic no sweat!

Reply:
I checked and found the issue.
Those files were indeed formatted by prettier on my IDE, but not by running `yarn lint`.
Actually, `yarn lint` itself was not targetting it, because seems that the glob pattern needs to be around quotes (https://prettier.io/docs/en/cli.html), but in package.json the quotes aren't present (https://github.com/ourjapanlife/findadoc-frontend/blob/main/package.json#L11).

To test, I tried to revert this specific file (`components/ContributorLead.vue`) to before the commit, and ran `prettier --write components/ContributorLead.vue` and it got formatted to the way I committed.
Then I reverted back again and ran `prettier --write "**/*.{js,vue,json,scss,md}"` (using double quotes around the pattern) and again it made the files to the way I committed.

So this seems to be correct?

Reply:
@ann-kilzer Please check this one also.
Let me know if I should revert the style, as it seems it was supposed to be this way all along but the `yarn lint` wasn't correctly configured.

Reply:
Can you please merge with the latest main, because I recently merged a fix to the lint commands. 🙇🏻‍♀️

Reply:
@ann-kilzer Sure, merged